core/drivers/crypto/se050/glue/include/i2c_native.h

```diff
@@ -9,8 +9,25 @@
 
 #include <kernel/rpc_io_i2c.h>
 
+#if defined(CFG_CORE_SE05X_I2C_TRAMPOLINE) && CFG_CORE_SE05X_I2C_TRAMPOLINE
+static inline TEE_Result native_i2c_transfer(struct rpc_i2c_request *req,
+			       size_t *bytes)
+{
+	/* Stub only */
+	(void)req;
+	(void)bytes;
+	return TEE_SUCCESS;
```

Comment:
prefer:
```c
static inline TEE_Result
native_i2c_transfer(struct rpc_i2c_request *req __unused,
		    size_t *bytes __unused)
{
	return TEE_SUCCESS;
}
```

`return TEE_ERROR_NBOT_SUPPORTED` would be even better, unless some side effect on caller functions falling into the `__no_return` case.